### DISCUSSION TITLE
Dynamic task is not retrying even thought in the task definition is set to 3

### DISCUSSION THREAD

**Discussion by missioncoding** [Netflix/conductor#3677] [Q&A]
I have a dynamic fork which failed in the first attempt but i observed that the spawned task is not retried even though in the task definition i have set it to 3.

any pointers or setting i need to set for fork and join tasks here?

Thanks

**Comment by v1r3n** [comment]
@missioncoding do you have a task definition created for the spawned tasks?

**Reply by missioncoding** [reply]
yes i got a task for creating dynamic inputs followed by a fork task and join..
